container/container.go

```diff
@@ -56,6 +56,12 @@ func (c Container) ImageID() string {
 	return c.imageInfo.ID
 }
 
+// ImageDigest returns the manifest digest. This is usually a single string,
+// but can be multiple or none at all.
+func (c Container) ImageDigest() []string {
```

Comment:
I'm suffering a little cognitive dissonance here with the plural vs non plural disconnect - any reason the method isn't also plural?

Reply:
Nope, fixed!